.ci/scripts/install-tools.bat

```diff
@@ -0,0 +1,10 @@
+set GOPATH=%WORKSPACE%
+set MAGEFILE_CACHE=%WORKSPACE%\.magefile
+set PATH=%WORKSPACE%\bin;C:\ProgramData\chocolatey\bin;%PATH%
+
+choco install curl -y --no-progress --skipdownloadcache
+mkdir %WORKSPACE%\bin
+curl -sL -o %WORKSPACE%\bin\gvm.exe https://github.com/andrewkroh/gvm/releases/download/v0.2.1/gvm-windows-amd64.exe
```

Comment:
Can we preinstall gvm into the image to improve reliability? I think the Windows images we have on beats-ci have it installed already.

Reply:
we can check before to try to install it

Reply:
```suggestion
where /q gvm
IF ERRORLEVEL 1 (
 curl -sL -o %WORKSPACE%\bin\gvm.exe https://github.com/andrewkroh/gvm/releases/download/v0.2.1/gvm-windows-amd64.exe
)
```